src/TenantSecurityClient.php

```diff
@@ -9,8 +9,15 @@
 
 class TenantSecurityClient
```

Comment:
I think we need a class comment on this describing overall what this object does. Otherwise it's not clear that it's important when you look at the phpdoc.

Reply:
Good catch. I took a stab at a comment, but let me know if you have suggestions for improvement.